test/network/network.node.js

```diff
@@ -134,7 +136,8 @@ describe('network', () => {
 
       bitswapMockB._receiveError = (err) => deferred.reject(err)
 
-      const { stream } = await p2pA.dialProtocol(p2pB.peerInfo, '/ipfs/bitswap/' + version.num)
+      // TODO: set addr
```

Comment:
@vasco-santos the address should be added. The previous test is causing this to pass but if that changes this will break